frontend/src/app/modules/boards/boards-root/boards-root.component.ts

```diff
@@ -8,7 +8,7 @@ import {BoardAssigneeActionService} from "core-app/modules/boards/board/board-ac
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/9e5b6b49777d229a9eba6e0fdc8f169f6da51b80/frontend/src/app/modules/boards/boards-root/boards-root.component.ts#L8"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>